functions-and-operators/aggregate-group-by-functions.md

```diff
@@ -62,6 +62,32 @@ In addition, TiDB also provides the following aggregate functions:
 
 Except for the `GROUP_CONCAT()` and `APPROX_PERCENTILE()` functions, all the preceding functions can serve as [Window functions](/functions-and-operators/window-functions.md).
 
++ `APPROX_COUNT_DISTINCT(expr)`
+
+    This function returns the approximate distinct count of `expr`. It uses `BJKST` algorithm and consumes less memory during simultaneous computation of cardinality for a large number of data sets whose cardinality has power law distribution. Moreover, it's very accurate for data sets with small cardinality and very efficient on CPU.
```

Comment:
Should we explain that its usage is almost same like `COUNT(DISTINCT)` ?

Reply:
> Should we explain that its usage is almost same like `COUNT(DISTINCT)` ?

done